libs/api/unit/src/lib/core/query/get-alert-groups-by-id.query.ts

```diff
@@ -0,0 +1,32 @@
+import { Inject } from '@nestjs/common';
+import { IQueryHandler, QueryHandler } from '@nestjs/cqrs';
+
+import { AlertGroupEntity } from '../entity/alert-group.entity';
+import {
+	ALERT_GROUP_REPOSITORY,
+	AlertGroupRepository,
+} from '../repository/alert-group.repository';
+
+export class GetAlertGroupsByIdQuery {
```

Comment:
Shouldn't this be named `GetAlertGroupByIdQuery` as it only returns max 1 alert group?

Reply:
you are absolutely correct, changed